plugins/modules/ios_logging_global.py

```diff
@@ -38,7 +38,9 @@
             description: Logging buffer size
             type: int
           severity: &severity
-            description: Logging severity level
+            description:
+              - Logging severity level
+              - Idempotancy is not maintained for this attribute as value not shown in running-config
```

Comment:
maybe this behavior is specific to the version we currently test with, let's add integration tests to check this nature, and add a comment there. Incase this fails the test in the future we would know to update the doc and fix the test.